config_samples/bsc/testnet/wsteth_testnet_config_ethereum.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contracts": {
+    "0x8B715EAf61A7DdF61C67d5D46687c796D1f47146": "ERC1967Proxy",
+    "0x607b139bfee21b2676ee664a237a70d737b9466e": "NttManager",
+    "0xF2bc73502283fcaC4b047dfE45366d8744daaC5B": "ERC1967Proxy",
+    "0xaf99eeb712511303f8b829707aafc5881a125771": "WormholeTransceiver",
```

Comment:
❗ On the testnet WormholeTransceiver.sol, Transceiver.sol, ITransceiver.sol have different code.

Reply:
Yeah, got catch, extra getters :-1: